### PR TITLE
Replace builtin icons

### PR DESCRIPTION
**Pull request by kai2ou**

### CODE REVIEW

**Comment by williambj1** [comment]
@kai2ou 

First of all, thank you very much for your Apple minimalism themed icons, really appreciated.

1. With your icons, I'm unable to change the icons while scanning, which is extremely strange, does it work for you?
1. We're still missing icons for `WIFI` and `AirPortOff`, are you planning on making those as well?

P.S. Could you please revert commit 030978c and open a separate PR for that? Or maybe you can leave it for us to handle since it's producing compilation errors. ATM I'm not sure how to implement icon highlight color reversing in macOS 10.13 and lower. `contentTintColor` seems to be the only solution. We may add `if #available(OSX 10.14, *) {}` as a temporary solution.

File: HeliPort.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -465,7 +465,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				MACOSX_DEPLOYMENT_TARGET = 10.15;
+				MACOSX_DEPLOYMENT_TARGET = 10.12;
```
Comment (williambj1) [inline HeliPort.xcodeproj/project.pbxproj:468]: Could you please revert this and open a separate PR for that?

File: HeliPort.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -520,7 +520,7 @@
...
```
Comment (williambj1) [inline HeliPort.xcodeproj/project.pbxproj:523]: Same as above

**Comment by kai2ou** [comment]
> d

**Comment by kai2ou** [comment]
> @kai2ou
> 
> First of all, thank you very much for your Apple minimalism themed icons, really appreciated.
> 
> 1. With your icons, I'm unable to change the icons while scanning, which is extremely strange, does it work for you?
> 2. We're still missing icons for `WIFI` and `AirPortOff`, are you planning on making those as well?
> 
> P.S. Could you please revert commit [030978c](https://github.com/zxystd/HeliPort/commit/030978c306fa62c2e6f930e0f685ba8effdaf3b1) and open a separate PR for that? Or maybe you can leave it for us to handle since it's producing compilation errors. ATM I'm not sure how to implement icon highlight color reversing in macOS 10.13 and lower. `contentTintColor` seems to be the only solution. We may add `if #available(OSX 10.14, *) {}` as a temporary solution.

It's my pleasure.
I put the files in here, [https://kai2ou.lanzous.com/iWR1Hda7cwj](url), you can handle.
I have no idea now for `WIFI` and `AirPortOff` too, I'll give it to you if I make.

**Pull request closed without merging**